src/resources/AzPowerShellSdkStrings.Designer.cs

```diff
@@ -0,0 +1,225 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
```

Comment:
For the PowerShell language worker, we do not checkin this file. Could you please remove it and regenerate it as part of the build process?

Reply:
Should I just include it in the .gitignore?

Reply:
I have included it in the .gitignore